### ISSUE TITLE
upload code for encoding transcripts

### ISSUE THREAD

**Issue by kmcdono2** [ohtap/ohtap#14]
### Summary

Ticket to arrange for transfer of @butter4fish code to encode transcripts into the repo.

### Actions

- [ ] assess where @hilarysun95 put the old code
- [ ] decide whether to make a new repo for encoding work
- [ ] introduce Yibing to this work

### Related Issues and Pull Requests

- #13 

### Updates

- **DATE OF UPDATE:** 
-

**Comment by butter4fish** [comment]
I think it would be nice to have all of the encoding scripts in one place. On the other hand, the way that you would run the scripts I wrote is pretty different from the way you would run Hillary's. (Mine uses a helper script to figure out who's talking and one of them requires some manual preprocessing to be done on the interviews) Would it be possible to reorganize the repo with Hillary's scripts so that there is a folder per collection? That might make the way that my scripts are organized a little more intuitive and leave space to explain how to run my scripts.

**Comment by Yibing-Du** [comment]
@kmcdono2 @butter4fish Shall we meet some time? I'd love to be introduced to the encoding task!

**Comment by butter4fish** [comment]
@Yibing-Du Yes! I think meeting would be a great idea! I'm available anytime after 9am est and the mornings usually work best for me. Should we switch over to email to set up a time?

**Comment by kmcdono2** [comment]
@Yibing-Du can you let me know how far work progressed on this?
